notebooks/summit/AuxTel/SITCOM-802/SITCOM-790_Sky_Flat_Sequence.ipynb

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
<p>This is the section I think we will want to automate with a verify_counts function, using the same method you defined above in the Find First Image section. So, after every image we should check the counts, and if they are not above our target value + some buffer (say 15000 counts instead of 10000), then we up the exposure time. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_isotuela/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>